tajo-core-tests/src/test/resources/results/TestInnerJoinWithSubQuery/testComplexJoinCondition5.result

```diff
@@ -5,25 +5,3 @@ n_nationkey,name1,name2
 2,BRAZ,BRAZ
 3,CANA,CANA
 4,EGYP,EGYP
-5,ETHI,ETHI
```

Comment:
Yes, because I changed the query. The original query performs the self-join on ```nation```, which makes the query plan non-deterministic. So, I changed it to the join of ```nation``` and ```small_nation``` which is derived from the ```nation``` table.